From 2e68292e40b6bbcb08d3080d33fd9d20cba8ec57 Mon Sep 17 00:00:00 2001
From: jinghua <xigu@marvell.com>
Date: Mon, 21 Aug 2017 22:22:04 +0800
Subject: [PATCH 2065/2241] fix: nas_init: a3700: remove affinity setting

Since Linux should be able to move tasks between two CPUs
based on their load in run-time, it is not required for
affinity setting anymore.

Change-Id: I8c03a45c570dcab8aba88872b9ffa035d5cfbebb
Signed-off-by: jinghua <xigu@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/43268
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Ofer Heifetz <oferh@marvell.com>
---
 tools/nas/nas_init.sh | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/tools/nas/nas_init.sh b/tools/nas/nas_init.sh
index 6cd1838..ae46e40 100755
--- a/tools/nas/nas_init.sh
+++ b/tools/nas/nas_init.sh
@@ -793,16 +793,6 @@ elif [ "$PLATFORM" == "a388" ]; then
 
 	set +o verbose
 	echo -ne "[Done]\n"
-elif [ "$PLATFORM" == "a37xx" ]; then
-	set -o verbose
-
-	# SATA
-	echo 2 > /proc/irq/32/smp_affinity
-	# PCI-E SATA controller
-	echo 2  > /proc/irq/34/smp_affinity
-
-	set +o verbose
-	echo -ne "[Done]\n"
 elif [ "$PLATFORM" == "a8040" ]; then
 	set -o verbose
 
-- 
2.7.4

